gramhopper/triggers/filter_triggers.py

```diff
@@ -33,32 +34,26 @@ def __init__(self, lang):
         super().__init__(Filters.language(lang))
 
 
+class _MessageTypeFilterBasedTrigger(FilterBasedTrigger):
+    def __init__(self, message_type=None):
+        try:
+            subfilters = message_type.split('.')
+            message_type_filter = reduce(
+                lambda filter_group, subfilter: getattr(filter_group, subfilter),
+                subfilters,
+                Filters
+            )
+            if isinstance(message_type_filter, BaseFilter):
+                super().__init__(message_type_filter)
+            else:
+                raise ValueError(f'"{message_type}" is not a valid message type to filter by, but it is a valid '
+                                 f'filter. Did you mean to use "{message_type}" as a filter type instead?')
+        except AttributeError:
+            raise ValueError(f'{message_type} is not a valid message type to filter by.')
```

Comment:
Please wrap `{message_type}` with `""` as you did in the other error message.

Reply:
mmmmmmkaaaaay